votes/2022-02-21-0.json

```diff
@@ -0,0 +1,149 @@
+      "Don't use any primordials (and undo existing ones) in the error path.": 0
+    },
+    "ronag": {
+      "Make the code in the error path full tamper proof by porting it away from JS/main thread (while we are waiting for a PR doing this, primordials should be used).": 3,
+      "Make the code in the error path full tamper proof by using primordials.": 0,
+      "Make  the code in the error path full tamper proof by using primordials + a build step to use bind-this proposal syntax.": 1,
+      "Use primordials in the error path only if there are no significant perf regression.": 2,
+      "Don't use primordials for prototype methods (and undo existing ones) in the error path, keep using them otherwise.": 4,
+      "Don't use any primordials (and undo existing ones) in the error path.": 5
+    },
+    "targos": {
+      "Make the code in the error path full tamper proof by porting it away from JS/main thread (while we are waiting for a PR doing this, primordials should be used).": 0,
+      "Make the code in the error path full tamper proof by using primordials.": 3,
+      "Make  the code in the error path full tamper proof by using primordials + a build step to use bind-this proposal syntax.": 3,
+      "Use primordials in the error path only if there are no significant perf regression.": 2,
+      "Don't use primordials for prototype methods (and undo existing ones) in the error path, keep using them otherwise.": 1,
+      "Don't use any primordials (and undo existing ones) in the error path.": 0
+    },
+    "tniessen": {
+      "Make the code in the error path full tamper proof by porting it away from JS/main thread (while we are waiting for a PR doing this, primordials should be used).": -1,
+      "Make the code in the error path full tamper proof by using primordials.": 0,
+      "Make  the code in the error path full tamper proof by using primordials + a build step to use bind-this proposal syntax.": -1,
+      "Use primordials in the error path only if there are no significant perf regression.": 0,
+      "Don't use primordials for prototype methods (and undo existing ones) in the error path, keep using them otherwise.": -1,
+      "Don't use any primordials (and undo existing ones) in the error path.": 0
+    },
+    "Trott": {
+      "Make the code in the error path full tamper proof by porting it away from JS/main thread (while we are waiting for a PR doing this, primordials should be used).": 0,
+      "Make the code in the error path full tamper proof by using primordials.": 0,
+      "Make  the code in the error path full tamper proof by using primordials + a build step to use bind-this proposal syntax.": 0,
+      "Use primordials in the error path only if there are no significant perf regression.": 0,
+      "Don't use primordials for prototype methods (and undo existing ones) in the error path, keep using them otherwise.": 0,
+      "Don't use any primordials (and undo existing ones) in the error path.": 0
+    }
+  }
+}
```

Comment:
I can't see my vote. can you double check to see where it went?

Reply:
Very sorry for this, it was a silly mistake of mine. Thanks for noticing, fixed in https://github.com/nodejs/TSC/pull/1158/commits/42b7c02c1c6ef334c33e55bfd0560d3d951c9eb7.